policy/modules/contrib/gpg.te

```diff
@@ -166,6 +166,7 @@ userdom_manage_user_home_content_files(gpg_t)
 userdom_manage_user_home_content_dirs(gpg_t)
 userdom_filetrans_home_content(gpg_t)
 userdom_stream_connect(gpg_t)
+userdom_watch_tmp_dirs(gpg_t)
```

Comment:
I wonder if this one won't be covered by `userdom_manage_all_user_tmp_content(gpg_t)` after #574 is merged.

Reply:
And indeed it is:
```
$ macro-expander 'userdom_manage_all_user_tmp_content(gpg_t)' | grep watch
allow gpg_t user_tmp_type:dir { create open getattr setattr read write link unlink rename search add_name remove_name reparent rmdir lock ioctl watch watch_reads };
allow gpg_t user_tmp_type:file { create open getattr setattr read write append rename link unlink ioctl lock watch watch_reads };
allow gpg_t user_tmp_type:lnk_file { create getattr setattr read write append rename link unlink ioctl lock watch watch_reads };
```